packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -28,6 +28,18 @@ import {
 
 const mockFetchSearchSuggestions = jest.fn();
 
+function getExpectedVisualTypeName( type ) {
+	const builtInLabels = {
+		post: 'post',
+		page: 'page',
+		post_tag: 'tag',
+		category: 'category',
+		attachment: 'attachment',
```

Comment:
```suggestion
		post: 'Post',
		page: 'Page',
		post_tag: 'Tag',
		category: 'Category',
		attachment: 'Attachment',
```

I think the labels should be capitalized.

Reply:
I trust folks won't mind if I reinstate my previous feedback that I don't think this PR is about capitalisation which is a design change. 

I really think it should remain focused on i18n and anything else should be a followup.  

The use of lowercase was a deliberate design decision and generally in the interface we are tending towards lower casing these. 

Sorry if that sounds pedantic but as a long time maintainer of this component I'm conscious we should not make these changes arbitrarily even if they seem logical.

Reply:
> The use of lowercase was a deliberate design decision and generally in the interface we are tending towards lower casing these.

In trunk, the labels are indeed lowercase, but they are capitalized by 'text-transform: capitalize;'. Do you mean to keep the labels lowercase and translatable, but also keep `text-transform: capitalize;`?

Reply:
I take it back. It seems everything is already in the correct case. You can ignore my previous comments! Apologies.